cg/src/domains/management/subjects/subjects.service.js

```diff
@@ -53,6 +53,39 @@ class SubjectsService {
 
     return decryptedSubjectsData;
   }
+
+  async listRectificationRequests(requestedPage) {
```

Comment:
you could also say:
requestedPage = 1
that would remove the if statement below